src/main/java/core/basesyntax/storage/ListStorageUser.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.storage;
+
+import core.basesyntax.model.User;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ListStorageUser {
+    public static List<User> userList = new ArrayList<>();
+
+    public void addUser(User user) {
+        userList.add(user);
+        System.out.println(user + " User added.");
+    }
```

Comment:
I think class ListStorageUser has only one function - to store data. It shouldn't do something with it. We have UserDao for it. And you have already implemented method addNewUser() in UserDao. So I suggest to delete addUser from ListStorageUser.

Reply:
Ok, for this situation. But in fruit shop I was told to move some logic to store, so I used that expirience.